### ISSUE TITLE
3.5.0rc3: error opening admin/instances page.

### ISSUE THREAD

**Issue by Izorkin** [mastodon/mastodon#17882]
### Steps to reproduce the problem

1. Set russian interface.
2. Open page "Настройки -> Модерация -> Федерация"

Error log:
```
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] method=GET path=/admin/instances format=html controller=Admin::InstancesController action=index status=500 error='ActionView::Template::Error: translation data {:one=>"%{count} известная учётная запись", :other=>"%{count} известных учётных записей", :zero=>"Нет известных учётных записей"} can not be used with :count => 5. key 'many' is missing.' duration=71.55 view=0.00 db=21.45
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] ActionView::Template::Error (translation data {:one=>"%{count} известная учётная запись", :other=>"%{count} известных учётных записей", :zero=>"Нет известных учётных записей"} can not be used with :count => 5. key 'many' is missing.):
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]     12:         - else
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]     13:           = t('admin.accounts.no_limits_imposed')
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]     14:
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]     15:     .trends__item__current{ title: t('admin.instances.known_accounts', count: instance.accounts_count) }= friendly_number_to_human instance.accounts_count
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580]
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] app/views/admin/instances/_instance.html.haml:15
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] app/views/admin/instances/_instance.html.haml:2
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] app/views/admin/instances/index.html.haml:46
puma[127644]: [c86a4667-0af2-4208-876d-66c519315580] app/controllers/concerns/localized.rb:11:in `set_locale'
```

### Expected behaviour

Correct opening admin/instances page.

### Actual behaviour

Error opening admin/instances page.

### Specifications

OS: NixOS, 22.05 (Quokka)
NnodeJS: v16.14.2
Ruby: 3.0.3

**Comment by ClearlyClaire** [comment]
This is caused by some Russian translations missing the proper pluralization keys. It's not limited to Russian, though there are a few other translations that have similar errors (which can be found using `i18n-tasks missing -t plural`, so that probably should be added to CI):

```
Missing translations (25) | i18n-tasks v0.9.37
+--------+--------------------------------------------------+----------------------------------+
| Locale | Key                                              | Value in other locales or source |
+--------+--------------------------------------------------+----------------------------------+
|   ar   | admin.instances.known_accounts                   | zero, two, few, many             |
|   gd   | admin.accounts.previous_strikes_description_html | two, few                         |
|   gd   | admin.instances.known_accounts                   | two, few                         |
|   gd   | admin.trends.links.shared_by_over_week           | two, few                         |
|   gd   | admin.trends.tags.used_by_over_week              | two, few                         |
|   hr   | invites.max_uses                                 | many                             |
|   hr   | statuses.poll.total_people                       | many                             |
|   hr   | statuses.poll.total_votes                        | many                             |
|   pl   | admin.accounts.previous_strikes_description_html | few, many                        |
|   ru   | admin.accounts.previous_strikes_description_html | few, many                        |
|   ru   | admin.instances.known_accounts                   | few, many                        |
|   ru   | admin.trends.links.shared_by_over_week           | few, many                        |
|   ru   | admin.trends.tags.used_by_over_week              | few, many                        |
|   sr   | about.status_count_after                         | many                             |
|   sr   | about.user_count_after                           | many                             |
|   sr   | accounts.followers                               | many                             |
|   sr   | accounts.posts                                   | many                             |
|   sr   | generic.validation_errors                        | many                             |
|   sr   | invites.max_uses                                 | many                             |
|   sr   | notification_mailer.digest.new_followers_summary | many                             |
|   sr   | notification_mailer.digest.subject               | many                             |
|   sr   | statuses.attached.image                          | many                             |
|   sr   | statuses.attached.video                          | many                             |
|   sr   | statuses.disallowed_hashtags                     | many                             |
|   uk   | admin.instances.known_accounts                   | few, many                        |
+--------+--------------------------------------------------+----------------------------------+
```

I'm not sure why those translations failed to be properly localized while other ones worked fine.

**Comment by Izorkin** [comment]
@ClearlyClaire thanks for help.
Small fix:
```diff
diff --git a/config/locales/ru.yml b/config/locales/ru.yml
index 4440f3336..eaa032fe3 100644
--- a/config/locales/ru.yml
+++ b/config/locales/ru.yml
@@ -506,7 +506,8 @@ ru:
       empty: Домены не найдены.
       known_accounts:
         one: "%{count} известная учётная запись"
-        other: "%{count} известных учётных записей"
+        few: "%{count} известных учётных записи"
+        many: "%{count} известных учётных записей"
         zero: Нет известных учётных записей
       moderation:
         all: Все

```

**Comment by ClearlyClaire** [comment]
Yeah, but it's not the only string, and upon investigation it seems it once worked. It was broken by 1761d9ba5d518a4e28ddc704b64efdba38d42063 for some reason…

This might be a Crowdin bug caused by changing source strings in 1abf0f90000c86bfbc5d6ac9a976834dcd17983a?

**Comment by Izorkin** [comment]
> This might be a Crowdin bug caused by changing source strings in #17576?

I can't check Crowdin.

**Comment by Izorkin** [comment]
Fixed in #17897.
